src/main/java/org/pegdown/ToHtmlSerializer.java

```diff
@@ -245,18 +243,18 @@ public void visit(SimpleNode node) {
                 throw new IllegalStateException();
         }
     }
-    
+
     public void visit(StrongEmphSuperNode node) {
-    	if(node.isClosed()){
-    		if(node.isStrong())
-    			printTag(node, "strong");
-    		else
-    			printTag(node, "em"); 
-    	} else {
-	    	//sequence was not closed, treat open chars as ordinary chars
-	    	printer.print(node.getChars());
-	    	visitChildren(node);
-    	}
+        if (node.isClosed()) {
```

Comment:
I checked and it is not in my source. I have all spaces. It would be difficult for me to have tabs, unless I copy/paste them in.

Best regards,


Vladimir.

> On Aug 22, 2015, at 5:38 PM, Mathias <notifications@github.com> wrote:
> 
> In src/main/java/org/pegdown/ToHtmlSerializer.java <https://github.com/sirthias/pegdown/pull/181#discussion_r37700041>:
> 
> >      public void visit(StrongEmphSuperNode node) {
> > -    	if(node.isClosed()){
> > -    		if(node.isStrong())
> > -    			printTag(node, "strong");
> > -    		else
> > -    			printTag(node, "em");
> > -    	} else {
> > -	    	//sequence was not closed, treat open chars as ordinary chars
> > -	    	printer.print(node.getChars());
> > -	    	visitChildren(node);
> > -    	}
> > +        if (node.isClosed()) {
> tabs?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/sirthias/pegdown/pull/181/files#r37700041>.
>